src/data/__tests__/projectIdSingleton.test.ts

```diff
@@ -7,12 +15,10 @@ describe('ProjectIdSingleton', () => {
 
   test('issues whoami requests for unknown projectId', async () => {
     // @ts-ignore
```

Comment:
**question:** Do we still need the `@ts-ignore` on these since we're no longer overriding `global`?

Reply:
We do because typescript doesn't seem to understand that the real implementation got replaced with a mock and `mockResolvedValue` isn't part of the real fetch interface.